src/elexmodel/handlers/data/BaseDataHandler.py

```diff
@@ -0,0 +1,60 @@
+import abc
+from io import StringIO
+from pathlib import Path
+
+import pandas as pd
+
+from elexmodel.handlers.data.Estimandizer import Estimandizer
+from elexmodel.utils.file_utils import get_directory_path
+
+
+class BaseDataHandler(abc.ABC):
+    """
+    Abstract base handler for model data
+    """
+
+    def __init__(
+        self, election_id, office, geographic_unit_type, estimands, s3_client=None, historical=False, data=None
+    ):
+        self.election_id = election_id
+        self.office = office
```

Comment:
I think this should be `office_id` rather than office, since that is how we reference it everywhere else

Reply:
Thanks, I agree!  I think I saw it as `self.office` in some of the data handlers and kept it lol 😄 🎉